models/branches.go

```diff
@@ -165,6 +165,20 @@ func (protectBranch *ProtectedBranch) GetGrantedApprovalsCount(pr *PullRequest)
 	return approvals
 }
 
+// GetRejectedReviewsCount returns the number of rejected reviews for pr.
+func (protectBranch *ProtectedBranch) GetRejectedReviewsCount(pr *PullRequest) int64 {
+	rejects, err := x.Where("issue_id = ?", pr.Issue.ID).
```

Comment:
can you add pr.LoadIssue() before?

Reply:
I wondering if would not be better use `pr.IssueID` instead of `pr.Issue.ID`, likewise the other methods.

Reply:
Yes, `pr.IssueID` is the way to go.